src/index.js

```diff
@@ -21,41 +54,11 @@ if (!window.cordova && window.location.protocol === 'file:') {
 
 // CSS.
 if (utils.device.isBrowserEnvironment) {
+  window.logs = debug;
   require('./style/aframe.css');
   require('./style/rStats.css');
```

Comment:
These are the only real `require` statements left in the `src/` folder. Not sure what the best way to get rid of them is, but that's probably best suited for a follow-up task.